src/include/storage/copier/npy_reader.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include "common/exception.h"
+#include "common/types/types.h"
+
+namespace kuzu {
+namespace storage {
+
+class NpyReader {
+
+public:
+    explicit NpyReader(const std::string& filePath);
+
+    ~NpyReader();
+
+    size_t getNumElementsPerRow() const;
+
+    void* getPointerToRow(size_t row) const;
+
+    inline std::string getFileName() const { return filePath; }
+
+    inline size_t getLength() const { return shape[0]; }
+
+    inline common::DataTypeID getType() const { return type; }
+
+    inline std::vector<size_t> const& getShape() const { return shape; }
+
+    inline size_t getNumDimensions() const { return shape.size(); }
+
+    inline size_t getMaxOffset() const { return fileSize - dataOffset; }
+
+private:
+    static inline bool isMachineLittleEndian() {
```

Comment:
We have a similar function in order by to check whether a machine is little endian or not.
Can you move this function to Utils.h? So that orderBy and npy_reader can use it.

Reply:
Moved